error in stacktrace.go

I get this error when I try to build my code
```
../../getsentry/raven-go/stacktrace.go:62:17: cannot convert f (type "github.com/pkg/errors".Frame) to type uintptr
```

--- a/stacktrace.go
+++ b/stacktrace.go
@@ -58,7 +58,7 @@ func GetOrNewStacktrace(err error, skip int, context int, appPackagePrefixes []s
 	})
 	if errHasStacktrace {
 		var frames []*StacktraceFrame
-		for _, f := range stacktracer.StackTrace() {
+		for f := range stacktracer.StackTrace() {
 			pc := uintptr(f) - 1
 			fn := runtime.FuncForPC(pc)
 			var fName string

